jobs/e2e_node/containerd/image-config-serial.yaml

```diff
@@ -1,11 +1,11 @@
 images:
   ubuntu:
-    image_family: pipeline-1-24
-    project: ubuntu-os-gke-cloud
+    image_family: ubuntu-2204-lts
+    project: ubuntu-os-cloud
     machine: n1-standard-2 # These tests need a lot of memory
     metadata: "user-data</workspace/test-infra/jobs/e2e_node/containerd/init.yaml,cni-template</workspace/test-infra/jobs/e2e_node/containerd/cni.template,containerd-config</workspace/test-infra/jobs/e2e_node/containerd/config.toml"
```

Comment:
These need to point to containerd config file that will sets `CONTAINERD_SYSTEMD_CGROUP` so that systemd cgroup driver will be configured on containerd side as well. I don't see that being the case currently?

Reply:
It looks like these jobs just use this helper script (https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/init.yaml) which installs this containerd config file - https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/config.toml

Do we need to maybe make a copy of that `config.toml` for systemd cgroup driver and point the test to use that config file?

Reply:
> The other option is to configure this test to also use containerd from main like the some of the other jobs, so we can use CONTAINERD_SYSTEMD_CGROUP setting.

What was the reason for these tests using separate config.toml file?  If we change to how its done in other tests, like you suggested, is there something else that need to be taken care of?

Reply:
I have created a new `config-systemd.toml` that will be used instead of `config.toml` to use systemd as cgroup driver.